modules/guardduty-member-account/outputs.tf

```
@@ -0,0 +1,14 @@
+output "guardduty_member_detector" {
```

We can remove the redundancy in the output names...

```suggestion
output "detector" {
```

---

Removed GuardDuty from all of the output names.